build.sh

```diff
@@ -28,25 +28,26 @@ HELP="$0 [<target> ...] [<flag> ...]
                       around the C++ code.
    cuml             - build the cuml Python package
    cpp-mgtests      - build libcuml mnmg tests. Builds MPI communicator, adding MPI as dependency.
-   prims            - build the ML prims tests
-   bench            - build the cuml C++ benchmark
+   prims            - build the ml-prims tests
+   bench            - build the libcuml C++ benchmark
    prims-bench      - build the ml-prims C++ benchmark
    cppdocs          - build the C++ API doxygen documentation
    pydocs           - build the general and Python API documentation
  and <flag> is:
    -v               - verbose build mode
    -g               - build for debug
    -n               - no install step
+   -h               - print this text
    --allgpuarch     - build for all supported GPU architectures
    --buildfaiss     - build faiss statically into libcuml
    --buildgtest     - build googletest library
```

Comment:
As mentioned in https://github.com/rapidsai/cuml/pull/3867#discussion_r633623933 .  This line should be removed